scripts/commands/addkeyitem.lua

```diff
@@ -11,32 +11,41 @@ cmdprops =
     parameters = "ss"
 };
 
+function error(player, msg)
+    player:PrintToPlayer(msg);
+    player:PrintToPlayer("@addkeyitem <ID> <player>");
+end;
+
 function onTrigger(player, keyId, target)
-    if (keyId == nil or keyId == 0) then
```

Comment:
was about to ask to see the crash dump, just realized why it would crash - checkin for zero when value could now possibly be a string. depr derp. It would hit debug break in luabaseentity if an invalid string got passed.